github_connector/models/github_repository_branch.py

```diff
@@ -192,15 +192,10 @@ def _download_code(self):
                     res = check_output(
                         ["git", "pull", "origin", branch.name], cwd=branch.local_path
                     )
-                    if branch.state == "to_download" or b"up-to-date" not in res:
-                        branch.write(
-                            {
-                                "last_download_date": datetime.today(),
-                                "state": "to_analyze",
-                            }
-                        )
-                    else:
-                        branch.write({"last_download_date": datetime.today()})
+                    vals = {"last_download_date": datetime.today()}
+                    if b"up to date" not in res:
```

Comment:
Have you checked that the result is in binary format?

Reply:
Response:

`b'Already up to date.\n'`